From 26a5a26280919ec42d1d69b0e94b7c8ab52e3bd8 Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@amd.com>
Date: Tue, 26 Oct 2021 12:14:42 -0500
Subject: [PATCH 43/58] SUNLIGHT: platform/x86: amd-pmc: Downgrade dev_info
 message to dev_dbg

Description:
 - For the majority of users this information will not be informative
   as they've chosen to program the RTC before going to sleep.

Bug: N/A
Change-Id: I5d1f21cc5c8b7f099257ec45fd19ad480b805882
Signed-off-by: Ionut Nechita <ionut_n2001@yahoo.com>
Suggested-by: Alexandre Belloni <alexandre.belloni@bootlin.com>
Signed-off-by: Mario Limonciello <mario.limonciello@amd.com>
---
 drivers/platform/x86/amd-pmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/x86/amd-pmc.c b/drivers/platform/x86/amd-pmc.c
index 1bd03f14a702..ffee7b6b6502 100644
--- a/drivers/platform/x86/amd-pmc.c
+++ b/drivers/platform/x86/amd-pmc.c
@@ -457,7 +457,7 @@ static int amd_pmc_verify_czn_rtc(struct amd_pmc_dev *pdev, u32 *arg)
 
 	*arg |= (duration << 16);
 	rc = rtc_alarm_irq_enable(rtc_device, 0);
-	dev_info(pdev->dev, "wakeup timer programmed for %lld seconds\n", duration);
+	dev_dbg(pdev->dev, "wakeup timer programmed for %lld seconds\n", duration);
 
 	return rc;
 }
-- 
2.35.1

